pandas/core/categorical.py

```diff
@@ -2013,7 +2014,20 @@ def repeat(self, repeats, *args, **kwargs):
 # The Series.cat accessor
 
 
-class CategoricalAccessor(PandasDelegate, NoNewAttributesMixin):
+@accessors.wrap_delegate_names(delegate=Categorical,
```

Comment:
I'm amenable to that.  Over in `core.indexes.accessors` there is a comment above `DatetimeAccessor` noting an alternative way to define the accessed properties/methods.

The only pattern I actively dislike is `CategoricalIndex._add_accessors`; pinning attributes to the class from a distance can lead to surprises (like the still-mysterious circular import `core.series`<-->`plotting._core`).